src/main/java/com/qaprosoft/carina/demo/mobile/gui/mfp/pages/common/CommonPageBase.java

```diff
@@ -10,4 +10,22 @@ public CommonPageBase(WebDriver driver) {
 
     public abstract boolean isUserLoggedIn(String username);
 
+    public abstract AbstractPage clickBottomNavigatorButton(BottomNavigatorButtons button);
+
+    public enum BottomNavigatorButtons{
+        DASHBOARD("Dashboard"),
+        DIARY("Diary"),
+        NEWSFEED("Newsfeed"),
+        PLANS("Plans");
+
+        String button;
```

Comment:
should be private

Reply:
Also we should know that value for ios and android could be different, so we need to split it. And this options return us some class - cauze it clickable, so class should be placed here as well
private String optionAndroid;
private String optionIos; and third private option is abstract class. Also you should create methods to get values for each option - ios value, android value and class.